lib/core/layout/components/sidenav-layout/sidenav-layout.component.ts

```diff
@@ -25,7 +37,8 @@ import { BehaviorSubject, Observable } from 'rxjs';
 @Component({
     selector: 'adf-sidenav-layout',
     templateUrl: './sidenav-layout.component.html',
-    styleUrls: ['./sidenav-layout.component.scss']
+    styleUrls: ['./sidenav-layout.component.scss'],
+    encapsulation: ViewEncapsulation.None
```

Comment:
Can we please also add host element class like all Material components have?

```ts
host: { class: 'adf-sidenav-layout' }
```

Will be much eaiser to style from outside apps.